tests/integration/model/test_models.py

```diff
@@ -24,6 +24,11 @@ def models() -> list[str]:
     return get_available_models()
 
 
+def frameworks() -> list[ExportType]:
```

Comment:
how about the one below to follow the existing terminology?
```suggestion
def export_types() -> list[ExportType]:
```